packages/firebase_auth/firebase_auth_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.4
+
+- Correct support for `confirmPasswordReset`
```

Comment:
```suggestion
- **Breaking change**: Added missing `app` parameter to `confirmPasswordReset`.
  (This is an exception to the usual policy of avoiding breaking changes since
  `confirmPasswordReset` is a new API and doesn't have clients yet.)
```